app/src/main/java/org/fossasia/openevent/general/event/subtopic/EventSubTopicIdConverter.kt

```diff
@@ -0,0 +1,16 @@
+package org.fossasia.openevent.general.event.subtopic
+
+import androidx.room.TypeConverter
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+
+class EventSubTopicIdConverter {
```

Comment:
@iamareebjamal here i save an embedded object into event model

Reply:
You have to add a type converter. Room does not know how to save that into DB

Reply:
this is an type converter